src/test/java/de/uhd/ifi/se/decision/management/jira/consistency/TestDuplicateFragment.java

```diff
@@ -0,0 +1,47 @@
+package de.uhd.ifi.se.decision.management.jira.consistency;
+
+import com.atlassian.jira.issue.Issue;
+import com.atlassian.jira.issue.MutableIssue;
+import com.atlassian.jira.project.Project;
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.consistency.implementation.DuplicateFragment;
+import de.uhd.ifi.se.decision.management.jira.testdata.JiraIssues;
+import de.uhd.ifi.se.decision.management.jira.testdata.JiraProjects;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class TestDuplicateFragment extends TestSetUp {
+
+	List<MutableIssue> testIssues;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)